packages/react-components/src/components/AppFrame/components/NavigationTopBar/types.ts

```diff
@@ -66,4 +75,9 @@ export interface ITopBarAlertProps extends ComponentCoreProps {
    * Show or hide the alert, defaults to `true`. Changes to this prop are animated - the alert will enter and leave smoothly.
    * */
   isVisible?: boolean;
+
+  /**
+   * The CSS class name
+   * */
+  className?: string;
```

Comment:
ComponentCoreProps already have the className prop, so it is unnecessary to define it here.